libs/anglify/src/modules/snackbar/_variables.scss

```diff
@@ -0,0 +1,3 @@
+$snackbar-background-color: var(--anglify-snackbar-background-color, var(--color-background));
```

Comment:
Is the `$snackbar-background-color` necessary? Because it is covered by the `$snackbar-container-color`.

Reply:
Yes because the high emphasis one has a transparency on it.